File: java/Aksesoris.java
public class Aksesoris extends PetShop { // mendefinisikan kelas Aksesoris yang merupakan turunan dari PetShop
    protected String bahan; // atribut untuk menyimpan bahan aksesoris
    protected String warna; // atribut untuk menyimpan warna aksesoris

    // konstruktor default, menginisialisasi atribut dengan nilai default
    public Aksesoris() {
        super();
        this.bahan = "";
        this.warna = "";
    }

    // konstruktor dengan parameter, menginisialisasi atribut dengan parameter yang
    // diberikan
    public Aksesoris(int id, String nama_produk, int harga_produk, int stok_produk, String kategori_produk,
            String bahan, String warna) {
        super(id, nama_produk, harga_produk, stok_produk, kategori_produk);
        this.bahan = bahan;
        this.warna = warna;
    }

    // metode setter untuk atribut bahan
    public void setBahan(String bahan) {
        this.bahan = bahan;
    }

    // metode setter untuk atribut warna
    public void setWarna(String warna) {
        this.warna = warna;
    }

    // metode getter untuk atribut bahan
    public String getBahan() {
        return bahan;
    }

    // metode getter untuk atribut warna
    public String getWarna() {
        return warna;
    }
}

File: python/Aksesoris.py
from PetShop import PetShop

class Aksesoris(PetShop):  # mendefinisikan kelas Aksesoris yang merupakan turunan dari PetShop
    def __init__(self, id=0, nama_produk="", harga_produk=0, stok_produk=0, kategori_produk="", bahan="", warna=""):  # konstruktor default, menginisialisasi atribut dengan nilai default
        super().__init__(id, nama_produk, harga_produk, stok_produk, kategori_produk)
        self.bahan = bahan  # atribut untuk menyimpan bahan aksesoris
        self.warna = warna  # atribut untuk menyimpan warna aksesoris

    # metode setter untuk atribut bahan
    def set_bahan(self, bahan):
        self.bahan = bahan

    # metode setter untuk atribut warna
    def set_warna(self, warna):
        self.warna = warna

    # metode getter untuk atribut bahan
    def get_bahan(self):
        return self.bahan

    # metode getter untuk atribut warna
    def get_warna(self):
        return self.warna